src/fsharp/vs/ServiceDeclarations.fs

```diff
@@ -1365,8 +1365,18 @@ type FSharpAccessibility(a:Accessibility, ?isProtected) =
 
 /// An intellisense declaration
 [<Sealed>]
-type FSharpDeclarationListItem(name: string, nameInCode: string, fullName: string, glyph: FSharpGlyph, info, isAttribute: bool, accessibility: FSharpAccessibility option,
-                               kind: CompletionItemKind, isOwnMember: bool, priority: int) =
+type FSharpDeclarationListItem
+    ( name          : string,
+      nameInCode    : string,
+      fullName      : string,
+      glyph         : FSharpGlyph,
+      info,
+      isAttribute   : bool,
+      accessibility : FSharpAccessibility option,
+      kind          : CompletionItemKind,
+      isOwnMember   : bool,
+      priority      : int
+    ) =
 
```

Comment:
(we discussed a bit on slack, trying to wrap up some key points from my perspective)

@vasily-kirichenko it is unfortunate that my PR was merged while you are working on same area, it is true that I should have reverted that formatting before making the PR.

I was asked to fix-up the formatting change, but I wasn't given the specifics (besides "we don't like comma in front because it is not seen in codebase").

We have to look at the situation and where we want the code to be:

* do we like dozen parameters on same (or only few lines)? does it make it easy to merge changes?
* do we like to pass complex expressions to method/function/constructor calls (such as this Choice1Of2, which I assume will keep changing until we give it a proper type)? does it make it easy to merge changes?

I'm also more conservative in compiler code itself, but this area of tooling, I believe, will be exposed to more contributions from more people, there is less effort/risk involved to shape it the way we ideally want it (easier to add / merge parameters, or document them, reducing overhead for reader on calls involving many expressions complex or not).

I didn't realize this code was part of FCS when I made the fix and that Don has to merge it frequently, although it is going under significant changes with recent work happening, so the reformatting won't hurt as the whole implementation is changing, it was bad luck that it was you who had conflict rather than me.

I like reviews, including the stylistic ones, for stylistic ones we should state in those the proposed formatting to reduce back & forth (I'd have reverted it if it was what was asked); in recent stylistic review I made, I gave what I'd like to see instead in addition to the remark.
    
I don't have clear guide line for stylistic review, but I feel those aren't an issue worth asking changes for:
* comma at beginning or end
* operator at beginning or end
* spacing around comma, parens
* vertical alignment

as long as the change goes in direction of better readability, even when they don't match our personal preference or follow strictly code around (code around is often unkept so not necessarily a good metric).

Of course concern of minimizing diff applies, and it doesn't take long for someone to state it (and get it fixed), and of course I think we are all happy to have more people looking at / reviewing and contributing changes.

I hope nobody is infuriated with me pursuing a bit the discussion, which will help collaboration...